ui/roll20_character_sheet_0.2.css

```diff
@@ -1,953 +1,1080 @@
 /* Styles that apply everywhere */
 
 input[type=text] {
-    padding: 1px 4px 1px 4px;
-    background-color: white;
-    height: 22px;
+  padding: 1px 4px 1px 4px;
+  background-color: white;
+  height: 22px;
 }
 
 input[type=number] {
-    padding: 1px 0px 1px 3px;
-    text-align: center;
-    background-color: white;
-    height: 22px;
+  padding: 1px 0px 1px 3px;
+  text-align: center;
+  background-color: white;
+  height: 22px;
 }
 
 [data-groupname] button:not([type="roll"]) {
-    padding: 3px 8px 3px 8px;
-    margin-top: 3px;
+  padding: 3px 8px 3px 8px;
+  margin-top: 3px;
 }
 
 select {
-    padding: 1px 4px 1px 4px;
-    height: 22px;
-    margin-bottom: 0px;
-    text-align: center;
-    text-align-last: center;
+  padding: 1px 4px 1px 4px;
+  height: 22px;
+  margin-bottom: 0px;
+  text-align: center;
+  text-align-last: center;
 }
 
 /* Change the character for dice roll buttons to be
    the two joined squares character and give them the blue background */
 button.btn.ui-draggable[type=roll] {
-    background-image: linear-gradient(90deg, #ddf, #99f);
+  background-image: linear-gradient(90deg, #ddf, #99f);
 }
+
```

Comment:
In VSCode, I ran the auto-formatter, which made these files more consistent.
Consider reviewing ignoring whitespace:
https://github.com/epic-power-rpg/roll20/pull/10/files?diff=unified&w=1